checkpoint/types/msg.go

```diff
@@ -18,6 +18,46 @@ import (
 
 var _ sdk.Msg = &MsgCheckpoint{}
 
+// MsgCheckpointAdjust represents checkpoint adjust
+type MsgCheckpointAdjust struct {
+	HeaderIndex uint64 `json:"header_index"`
+}
```

Comment:
This has to be the same as `MsgCheckpointAck` because we need to use msg value to store it in db in `MsgPostHandlerCheckpointAdjust`

```suggestion
// MsgCheckpointAdjust represents checkpoint adjust
type MsgCheckpointAdjust struct {
	From       types.HeimdallAddress `json:"from"`
	Number     uint64                `json:"number"`
	Proposer   types.HeimdallAddress `json:"proposer"`
	StartBlock uint64                `json:"start_block"`
	EndBlock   uint64                `json:"end_block"`
	RootHash   types.HeimdallHash    `json:"root_hash"`
	TxHash     types.HeimdallHash    `json:"tx_hash"`
	LogIndex   uint64                `json:"log_index"`
}
```